pkg/build/registry/buildconfiginstantiate/rest.go

```diff
@@ -241,7 +241,7 @@ func (h *binaryInstantiateHandler) handle(r io.Reader) (runtime.Object, error) {
 	if err != nil {
 		return nil, errors.NewInternalError(fmt.Errorf("unable to connect to node, could not retrieve TLS client config: %v", err))
 	}
-	upgrader := spdy.NewRoundTripper(tlsClientConfig)
+	upgrader := spdy.NewRoundTripper(tlsClientConfig, false)
```

Comment:
> what's the question/what does false represent?

FollowRedirects.  You don't follow all of your upstream dependencies?  I'm shocked! :)